app/testOnly/models/formsmodel/StartJourneyForm.scala

```diff
@@ -159,6 +161,7 @@ object StartJourneyForm {
           case TaxRegime.Epaye => payeTaxReferenceKey -> RandomDataGenerator.nextEpayeRefs()(Random)._3
           case TaxRegime.Vat   => vatTaxReferenceKey -> RandomDataGenerator.nextVrn()(Random)
           case TaxRegime.Sa    => saTaxReferenceKey -> RandomDataGenerator.nextSaUtr()(Random)
+          case TaxRegime.Sia   => saTaxReferenceKey -> RandomDataGenerator.nextSaUtr()(Random)
```

Comment:
this should be referring to NINO's instead of SA UTR. The stub will probably also need to be updated as well.
I think you'll need to add NINO in the form object and then you'd need to go into `TestUser`  and make sure we're creating a user there with the supplied NINO